### PR TITLE
KFSPTS-16446 add new fields to the RASS integration

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

**Comment by db483** [comment]
retest this please

**Comment by db483** [comment]
Please hold on reviewing this one, Jay has noted that I need to add truncating for sponsor award number now that his change is in.

**Comment by cah292** [comment]
@db483 Can you please confirm if this is ready for review now? I see that there was a recent commit to implement the needed truncation.

**Comment by db483** [comment]
Chad,

Thank you for checking. I've added the change but I still need to do some local validation. I will comment when this is ready.

**Comment by db483** [comment]
Chad,

This is ready for review now.

Thank you.

**Comment by db483** [comment]
This is ready for review again. Please note there is also a SQL PR.

File: src/test/java/edu/cornell/kfs/rass/batch/xml/fixture/RassXmlAwardEntryFixture.java [hunk 1]
```diff
@@ -188,6 +208,13 @@ public RassXmlAwardEntry toRassXmlAwardEntry() {
         award.setOrganizationCode(organizationCode);
         award.setCostShareRequired(costShareRequired);
         award.setFinalReportDueDate(buildDateFromDateTime(finalReportDueDate));
+        award.setEverify(everify);
+        award.setFinalFinancialReportRequired(finalFinancialReportRequired);
+        award.setBudgetStartDate(buildDateFromDateTime(budgetStartDate));
+        award.setBudgetStopDate(buildDateFromDateTime(budgetStopDate));
+        award.setPricingType(pricingType);
+        award.setPrimeAgreementNumber(primeAgreementNumber);
```
Comment (cah292) [inline src/test/java/edu/cornell/kfs/rass/batch/xml/fixture/RassXmlAwardEntryFixture.java]: The code above doesn't appear to be setting the budget total on the XML Award DTO.

**Comment by db483** [comment]
This is ready for review. Thank you.

**Pull request merged**